prisma/migrations/20240415113539_withdrawal_wallet_id/migration.sql

```diff
@@ -0,0 +1,38 @@
+-- AlterTable
+ALTER TABLE "Withdrawl" ADD COLUMN     "walletId" INTEGER;
+
+-- AddForeignKey
+ALTER TABLE "Withdrawl" ADD CONSTRAINT "Withdrawl_walletId_fkey" FOREIGN KEY ("walletId") REFERENCES "Wallet"("id") ON DELETE SET NULL ON UPDATE CASCADE;
+
+CREATE OR REPLACE FUNCTION create_withdrawl(lnd_id TEXT, invoice TEXT, msats_amount BIGINT, msats_max_fee BIGINT, username TEXT, auto_withdraw BOOLEAN, wallet_id INTEGER)
```

Comment:
I didn't replace the old function signature `TEXT, TEXT, BIGINT, BIGINT, TEXT, BOOLEAN` with `TEXT, TEXT, BIGINT, BIGINT, TEXT, INTEGER` since then the old code might be confused which function it should use if two signatures exist with the same amount of parameters during migration even though it should be obvious when looking at the JS types (boolean is not an integer).

But I neither trust JS nor Prisma to not fuck this up so to be safe, I added a new function signature.